### PR TITLE
Added Command to Get Route Data

### PR DESCRIPTION
**Pull request by deepak07031999**
This PR updates README.md file.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -72,6 +72,26 @@ node   ClusterIP   172.30.126.228   <none>        3000/TCP    1m
...
+                           exposed on router default (host apps-crc.testing) about a minute ago
+Path:                   <none>
+TLS Termination:        edge
+Insecure Policy:        <none>
+Endpoint Port:          3000
+
+Service:        node
+Weight:         100 (100%)
+Endpoints:      10.217.0.87:3000
+```
```
Comment (krishvoor) [inline README.md:94]: In this context, why would we need to do this?
I think this would be sufficient enough `oc get route -n <NAMESPACE>`

**Review by krishvoor — COMMENTED** [review]
Minor nits, otherwise /lgtm

File: README.md (hunk 1 above)
Comment (deepak07031999) [reply README.md:94]: Yes, oc get route -n <NAMESPACE> is sufficient ,  After that I have shown the result of that command.

Comment (krishvoor) [reply README.md:94]: FWIW, we do not need to `oc describe route -n <NAMESPACE>` in the context of deploying MongoDB+NodeJS application on OCP, beats the purpose. please update your PR.

**Comment by deepak07031999** [comment]
I have now removed oc describe route -n <NAMESPACE> command

**Pull request merged**